jenkins/check-for-build.jenkinsfile

```diff
@@ -71,7 +71,8 @@ pipeline {
                             echo "Skipping, ${sha.path} already exists."
                         } else {
                             try {
-                                // Currently only opensearch build takes an additional test manifest parameter
+                                // TEST_MANIFEST param isn't applicable for distribution-build-opensearch-dashboards job yet,
+                                // so we don't pass it to that job to prevent confusion from Jenkins UI
```

Comment:
Could we just add the param (and not use it) to that job and avoid the error entirely?

Reply:
Removed if/else logic, and passing the same params regardless of job.